src/scss/share.scss

```diff
@@ -105,20 +77,46 @@
 	$hover-color: str-slice(ie-hex-str($hover-color), 4);
 	$tint: "&tint=%23#{$hover-color},%23#{$hover-color}";
 
+	.#{$classname}__icon {
+		display: block;
+		position: relative;
+		top: -1px;
+		left: -1px;
+		width: $o-share-icon-size;
+		height: $o-share-icon-size;
+		border: 0;
+		cursor: pointer;
+
+		// Increase the hit target to improve a11y
+		&:after {
+			position: absolute;
+			display: block;
+			content: '';
+			width: $o-share-icon-size + 5;
+			height: $o-share-icon-size + 5;
+			top: -2px;
+			left: -2px;
+
+			@include oGridRespondTo(S) {
+				width: $o-share-icon-size + 10px;
+				height: $o-share-icon-size + 10px;
+				top: -5px;
+				left: -5px;
+			}
+		}
+	}
+
 	// scss-lint:disable DuplicateProperty
-	.#{$classname}__action--#{$icon-name} i {
+	.#{$classname}__icon--#{$icon-name}:before {
+		content: '';
+		display: block;
+		width: #{$o-share-icon-size};
+		height: #{$o-share-icon-size};
 		background-image: url($service-url + $query + "&format=svg");
 
 		// ie7/8 fallback. Uses the `\9` selector hack to target ie6-8.
 		// Hack is documented here: http://browserhacks.com/#hack-ab1bcc7b3a6544c99260f7608f8e845e
 		background-image: url($service-url + $query + "&format=png&width=#{$o-share-icon-size}")\9;
-
-		// Hack to load icon early to prevent FOIC on hover
```

Comment:
Why have you decided to remove this?

Reply:
Due to a lack of concentration, I didn't mean to remove it I was just moving some lines around and must have forgot to put this back in. Thanks for catching it